mercury-api/src/main/java/com/redhat/mercury/operator/model/ApiVersion.java

```diff
@@ -0,0 +1,13 @@
+package com.redhat.mercury.operator.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import lombok.ToString;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonDeserialize()
+@ToString
+public enum ApiVersion {
+    v1;
```

Comment:
As I said before, I'd better have this as a `String` constant instead of an `enum`